test-data/unit/check-modules.test

```diff
@@ -1640,3 +1640,9 @@ m = n  # E: Cannot assign multiple modules to name 'm' without explicit 'types.M
 [file n.py]
 
 [builtins fixtures/module.pyi]
+
+[case testPackageImportInCwd]
+from .m import *
```

Comment:
Hmm this might be more subtle than this. We should at least test that the module ids of the main module and `.m` are reasonable, and that all ways ways of referring to `m` work consistently. In particular:

* What is the fully-qualified module id of the main module and `m`?
* What would having `from m import *` in the main module result in (when there is `__init__.py` in the same directory as in this test case)?
* What is the module search path within mypy -- does it include the directory containing `__init__.py` or the parent directory? There might be no simple direct way to test this.